CHANGELOG.md

```diff
@@ -34,6 +34,8 @@
 
 **Changed**:
 
+- **decidim-initiatives**: For consistency with DB, use Ruby Dates instead of DateTimes, rename signature_(start/end)_time fields to signature_(start/end)_date. [\#3932](https://github.com/decidim/decidim/pull/3932)
```

Comment:
Can you quote the renamed fields, and maybe split them for clarity? Otherwise the rest of the changelog is displayed italicized because of the underscores.